haskell_101/codelab/README.md

```diff
@@ -0,0 +1,126 @@
+# Haskell 101 Codelab
+
+## Setup
+
+To setup your machine please consult [the official haskell website](https://www.haskell.org/platform/). We have also provided a docker container that can be used to run the test suite. This method for running the tests requires [Docker](https://www.docker.com/get-started) and [Docker compose](https://docs.docker.com/compose/install/) but avoids the need to set up haskell on your machine.
+
+## Running tests
+
+To run the tests first build the binary `make codelab` and then run the binary `./codelab` and you should see output similar to below as you progress through the lab.
+```
+### Setction 1
+add       1 2                             [OK] got: 3
```

Comment:
Let's remove the specific output.  Or at least most of it, maybe keeping one line.
Otherwise, we would need to keep this output in sync with the actual output.
It is more tedious than useful, IMHO.

Reply:
To clarify, are you only interested in the output from the build?

Reply:
👍 Sounds good, I will update this.